features/support/pages/my_projects_page.rb

```diff
@@ -0,0 +1,32 @@
+# Encapsulates user interactions with the My Projects Page
+class MyProjectsPage < SitePrism::Page
+  set_url "/my_projects"
+
+  elements :_pending_projects, "*[data-type='pending-projects'] *[data-type=project]"
+  elements :_approved_projects, "*[data-type='approved-projects'] *[data-type=project]"
```

Comment:
There's probably a better way to do this with sections or some such nonsense, but this worked pretty well.